ereolen_support/ereolen_support.helper.inc

```diff
@@ -0,0 +1,255 @@
+<?php
+
+/**
+ * @file
+ * Contains helper functions for the support form.
+ */
+
+/**
+ * Search for a drupal user email in Jira.
+ *
+ * @param \stdClass $user
+ *   A drupal user.
+ *
+ * @return bool|mixed
+ *   A jira user or FALSE if no user was found.
+ *
+ * @throws \Exception
+ */
+function _ereolen_support_search_jira_user($user, $form_state) {
+  $form_email = $form_state['values']['ereolen_support_email'];
+  $requestParts = _ereolen_support_get_request_parts('GET');
+  $result = drupal_http_request($requestParts['jira_path'] . '/user/search?includeInactive=true&username=' . $form_email, $requestParts['options']);
```

Comment:
`$form_email` should be validate to be an e-mail address to protect injection into jira. Jira should do this, but we are not always running the newest... so please use reg-ex for validation before sending data into jira.